src/auro-pane.js

```diff
@@ -85,9 +85,14 @@ class AuroPane extends LitElement {
       const priceClasses = {
         'price': true,
         'price--long': this.price.length > 6,
-        'child': true
+        'child': true,
+        'price--empty': this.price === ''
       };
 
+      if (this.price === '') {
+        this.price = '--';
+      }
```

Comment:
`this.price` is not set with a default value in the API. Does this need handle `this.price === undefined` or perhaps we need to default this.price to equal an empty string?

If we are going to support undefined we need a test for that scenario as well.

Reply:
I believe that we do support `this.price === undefined` with the "Date only" `auro-pane`. A basic test without a price is already in the test file.

Reply:
Doesn't that mean that line 92 should be `this.price === '' || this.price === undefined`?

Reply:
No. We want to only use `this.price === ''` because the "Dates only" should not display the "--" in the UI when `price === undefined`.